modules/markup/html_internal_test.go

```diff
@@ -0,0 +1,381 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+package markup
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/util"
+	"github.com/stretchr/testify/assert"
```

Comment:
Missing empty line between local and external package.

See https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide

Reply:
Addressed.